site/content/docs/guide/plugin/policy-expr.md

```diff
@@ -0,0 +1,193 @@
+---
+Policy Expression Reference
+---
+
+"Policy expressions" are a small expression language in Hipcheck that allows the
+JSON data output by analysis plugins to be reduced to a boolean pass/fail
+determination used for scoring. Policy expressions are mostly found in policy
+files, as the `policy` node for analyses or the `investigate` node for the
+entire analysis. Plugin authors may also want to be familiar with policy
+expressions, as one of the gRPC calls they may implement returns a default
+policy expression for the analysis implemented by the plugin.
+
+The policy expression language is limited. It does not permit user-defined
+functions, assignment to variables, or the retention of any state. Any policy
+expression supplied in a policy file which does not result in a boolean output
+will produce an error.
+
+If the policy expression language is insufficient to represent a desired policy
+on the output of a given plugin, users are encouraged to write their own plugin
+which takes as input that plugin's output and performs the desired manipulation.
+
+### Primitives
+
+Policy expressions have the following primitive types:
+
+| Type    | Description | Example |
+| ------- | ----------- | ------- |
+| integer | A signed 64-bit integer | `-5`, `360` |
+| float | A 64-bit float, NaN is disallowed | `2.001` |
+| boolean | A true or false value | `#t`, `#f` |
+| identifier | A function name or placeholder value in a lambda function | `add` |
+| datetime | A datetime value with timezone information. [More info](#datetime) | `2024-09-17T09:00-05` |
+| span | a (uniform) duration of time. [More info](#span) | `P5wT1h30m` |
+
+#### Datetime
+
+Datetimes use the string format from the `jiff` [crate](), which is a lightly
+modified version of ISO8601. A datetime must include a date in the format
+`<YYYY>-<MM>-<DD>`. An optional time in the format `T<HH>:[MM]:[SS]` will be
+accepted after the date. Decimal fractions of hours and minutes are not allowed;
+use smaller time units instead (e.g. `T10:30` instead of `T10.5`). Decimal
+fractions of seconds are allowed. The timezone is always internally resolved to
+UTC, but you can specify the datetime's original timezone as an an offset from
+UTC by including `+{HH}:[MM]` or `-{HH}:[MM]`.
+
+#### Span
+
+Spans represent a duration of time using the `jiff` [crate] `Span` type. Policy
+expression spans can include weeks, days, hours, minutes, and seconds (including
+decimal fractions of a second). Spans are prefixed with the letter "P" followed
```

Comment:
Change to something like "and can include optional decimal fractions of the smallest unit of time (hours, minutes, or seconds) used."